frontend/javascripts/oxalis/model/reducers/volumetracing_reducer_helpers.ts

```diff
@@ -179,6 +182,23 @@ export function setMappingNameReducer(
   if (mappingType !== "HDF5" || !isMappingEnabled) {
     mappingName = null;
   }
+  if (mappingName) {
+    const { agglomerates, mappings } = getSegmentationLayerByName(
+      state.dataset,
+      volumeTracing.tracingId,
+    );
+    const listToSearchIn = mappingType === "HDF5" ? agglomerates : mappings;
```

Comment:
Did you deliberately include both cases here for the code to be future-proof? If yes, that's appreciated :) I'm asking because in line 183 the mappingName is set to null for non-hdf5 mappings and this code is only executed if mappingName is not null (line 185).